packages/account/package.json

```diff
@@ -51,6 +51,9 @@
     "@biconomy/bundler": "^3.1.2",
     "@biconomy/modules": "^3.1.2",
     "@biconomy/paymaster": "^3.1.2",
+    "@ethersproject/abstract-signer": "^5.7.0",
+    "@ethersproject/providers": "^5.7.2",
+    "@ethersproject/wallet": "^5.7.0",
```

Comment:
We are only using the `ethersproject/providers` in tests. Can we remove this from dependency. If still needed should be there in dev dependency right?